src/nvim/api/vim.c

```diff
@@ -33,6 +34,22 @@
 # include "api/vim.c.generated.h"
 #endif
 
+/// @see foldCreate
```

Comment:
Tests are planned, that's why I started with https://github.com/neovim/neovim/pull/5993 .
If you could check it out that would be very cool.